.cookiecutter/includes/requirements/prod.in

```diff
@@ -18,11 +18,9 @@ pyramid-googleauth
 pyramid-jinja2
 pyramid-retry
 pyramid-services
-pyramid-tm
 requests
 requests-oauthlib
 sqlparse
 tabulate
 webargs
 xmltodict
-zope.sqlalchemy
```

Comment:
The cookiecutter adds `pyramid-tm` and `zope.sqlalchemy` now so LMS no longer needs them here